empire-db/src/test/java/org/apache/empire/commons/ObjectUtilsTest.java

```diff
@@ -52,6 +53,21 @@ public void testIsEmpty()
 		assertFalse(ObjectUtils.isEmpty(new Object()));
 	}
 
+	@Test
+    public void testIsZero()
+    {
+        assertTrue(ObjectUtils.isZero(BigDecimal.ZERO));
+        assertTrue(ObjectUtils.isZero(0f));
+        assertTrue(ObjectUtils.isZero(Float.valueOf("0")));
+        assertTrue(ObjectUtils.isZero(0d));
+        assertTrue(ObjectUtils.isZero(0l));
+        assertTrue(ObjectUtils.isZero(0));
+		assertTrue(ObjectUtils.isZero(null));
```

Comment:
please use tabs or spaces - not both

Reply:
oh, that was not intended.
Thanks i fixed it.